mysite/settings.py

```diff
@@ -11,7 +11,8 @@
 """
 
 import os
-import django_heroku
+
+# import django_heroku
```

Comment:
Why did you comment this out?

Reply:
This was just to work locally. Forgot to uncomment.